### DISCUSSION TITLE
cannot open draft issues on Projects

### DISCUSSION THREAD

**Discussion by dongsupark** [community/community#16083] [Projects and Issues] [answered]
Recently I am not able to open a draft issue on specific GitHub Projects.
Clicking on a draft issue results in opening a new browser tab, instead of showing the draft contents.

I was wondering if it was a web browser issue, so have tried doing it with both Firefox and Chrome, but nothing worked.
Could it be an issue on the GitHub side?

**Comment by radao** [comment]
Yup same problem. Worked fine yesterday. Checked a variety of browsers/OS.

**Comment by ch1bo** [comment]
Stopped working for us as well.

**Comment by DustyReagan** [comment]
Same issue here on Firefox and Safari. Issue must be on GitHub side.

**Comment by ch1bo** [comment]
Feedback items about this: #16083, #16093, #16103 (at least)

**Comment by ch1bo** [comment, marked as answer]
Enabling the feature preview setting (again?) seems to be a solution to this! https://github.com/github/feedback/discussions/16136#discussioncomment-2699638

**Comment by dongsupark** [comment]
@ch1bo Thanks for the info.
Indeed it works well after enabling `Project Side Panel (Alpha)` from Feature Preview settings.
